tests/integration/models/application.spec.coffee

```diff
@@ -142,14 +142,18 @@ describe 'Application Model', ->
 					promise = balena.models.application.hasAny()
 					m.chai.expect(promise).to.eventually.be.true
 
-			describe 'balena.models.application.create()', ->
+			describe.only 'balena.models.application.create()', ->
 
 				it 'should reject if trying to create an app with the same name', ->
 					promise = balena.models.application.create
 						name: 'FooBar'
 						applicationType: 'microservices-starter'
 						deviceType: 'beaglebone-black'
-					m.chai.expect(promise).to.be.rejectedWith('Application name must be unique')
+					m.chai.expect(promise).to.be.rejected
+					.then (error) ->
+						m.chai.expect(error).to.have.property('code', 'BalenaRequestError')
+						m.chai.expect(error).to.have.property('statusCode', 404)
```

Comment:
Why is it 404?

Reply:
This goes way back on how pine was originally statusing db & rule errors and we have kept if for compatibility even in v5. We might change this to a 409 in v6 though, since pine now property uses that status code.